internal/clients/http/ec2/ec2_client.go

```diff
@@ -403,6 +403,10 @@ func (c *ec2Client) RunInstances(ctx context.Context, params *clients.AWSInstanc
 						Key:   ptr.To("Name"),
 						Value: name,
 					},
+					{
+						Key:   ptr.To("rhhc:rid"),
+						Value: ptr.To(fmt.Sprintf("%s-%d", config.EnvironmentPrefix(2), reservation.ID)),
```

Comment:
I'd question the use of putting the environment in the tag, if the main target for this feature are our Customers, they do not care much about environment as that should always be prod. Or is there another reason you want to include environment?
I know it's annoying to deal with instances from stage, but won't that be weird to customers to have "pr-....." everywhere?